src/test/resources/hw1/AllTestSuite.xml

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+<suite name="All test suite">
+    <test name="CalculatorSubtractTest">
+        <classes>
+            <class name="hw1.CalculatorSubtractTest"></class>
+        </classes>
+    </test>
+
+    <test name="CalculatorSumTest">
+        <classes>
+            <class name="hw1.CalculatorSumTest"></class>
+        </classes>
+    </test>
+
+    <test name="CalculatorDivideTest">
+        <classes>
+            <class name="hw1.CalculatorDivideTest"></class>
+        </classes>
+    </test>
+
+    <test name="CalculatorMultiplyTest">
+        <classes>
+            <class name="hw1.CalculatorMultiplyTest"></class>
+        </classes>
+    </test>
+</suite>
```

Comment:
Why do you decide include set of all classes to common suite? Do we have other way add all tests to the main suite?

Reply:
Because we should create a test suite, which could run all the tests, and I added all tests to this suite. Also, we can create a test suite using groups or package.